src/script/interpreter.cpp

```diff
@@ -467,7 +467,9 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
                 opcode == OP_DIV ||
                 opcode == OP_MOD ||
                 opcode == OP_LSHIFT ||
-                opcode == OP_RSHIFT)
+                opcode == OP_RSHIFT ||
+                opcode == OP_VERIF ||
+                opcode == OP_VERNOTIF)
```

Comment:
`OP_VER` can be included here too:

```suggestion
                opcode == OP_VERNOTIF ||
                (fExec && opcode == OP_VER))
```

Reply:
cool, I've implemented!